### PR TITLE
[FIX] sale: fix the title of the SO card in the portal

### PR DESCRIPTION
**Pull request by ltinel**

### CODE REVIEW

File: addons/sale/i18n/sale.pot [hunk 1]
```diff
@@ -1634,9 +1634,7 @@ msgstr ""
 #: model:ir.model.fields,help:sale.field_sale_order__expected_date
 msgid ""
 "Delivery date you can promise to the customer, computed from the minimum "
-"lead time of the order lines in case of Service products. In case of "
-"shipping, the shipping policy of the order will be taken into account to "
-"either use the minimum or maximum lead time of the order lines."
```
[the remark this answers is not in the document]
Comment (ltinel) [reply addons/sale/i18n/sale.pot:1639]: Sure, I'll revert the auto-generated changes that are unrelated to my changes (however I don't think they're harmful. On the contrary, I believe that if some translations are overridden, these overrides should be handled in the overriding modules, not here)

[the remark this answers is not in the document]
Comment (Feyensv) [reply addons/sale/i18n/sale.pot]: yeah, that's what I meant with the 'current issues in the management', but they'll come back whenever someone reexports the pot from a full database (which is the flow followed by the translation team).
So I agree it's very probably not working fine (or you get wrong texts) if you don't have the overriding modules, but that's how it is everywhere right now.

File: addons/sale/i18n/sale.pot [hunk 2]
```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 17.0\n"
+"Project-Id-Version: Odoo Server 17.0+e\n"
```
Comment (Feyensv) [inline addons/sale/i18n/sale.pot:7]: This is also wrong, means you exported the pot with enterprise installed :eyes:
You can remove the changes in the file headers, I was never told they were necessary, and they'll bring conflicts in every forward-port :crying_cat_face:

**Review by Feyensv — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**